README.md

```diff
@@ -0,0 +1,91 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Personal Portfolio project
+
+## Built With
+
+- HTML/CSS
+- NPM, Git, Github
+
+## Getting Started
+
+**This project includes linters for Reporting errors in HTML, CSS and An open-source, automated tool for improving the quality of web pages, which has audits for performance, accessibility, progressive web apps, SEO and more. It is a summary of all my works and known technologies.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+1. Node package Manager(NPM)
+   - To install NPM on your local machine, please visit [ NPM official website](https://nodejs.org/en/download/)
+
+- Text editor(Vscode, Atom, NotePad++, Sublime)
+
+### Setup
+
+Set up folder as:
+
+1. /PORTFOLIO-MICROVERSE -.github/workflows - linters.yml
+   - images
+     - Ellipse 18.svg
+     - Ellipse 18@2x.png
+     - Ellipse 18.png
+     - Ellipse 18.svg
+     - github.png
+     - hamburger.png
+     - twitter.png
+     - Vector.png
+     - vector2.png
+     - vector3,png
+   - .hintrc
+   - .stylelintrc.json
+   - index.html
+   - LICENSE
+   - Package-lock.json
+   - Package.json
+   - style.css
+
+### Install
+
+Run the following commands to install dependencies
+
+```
+
+npm install --save-dev hint@7.x
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+```
+
+### Usage
+
+```
+
+npx hint .
+npx stylelint "**/*.{css,scss}"
+
+```
+
+### Deployment
+
+## Authors
+
+👤 **Omachi John Sunday**
+
+- GitHub: [@githubhandle](https://github.com/MrOmachi)
+- Twitter: [@twitterhandle](https://twitter.com/Mr_Omachi)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/john-omachi-00446210b)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] I suggest adding `MIT.md` file to your project. you can use [Microverse template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) and don't forget to customize it 👌.

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.

Reply:
Added MIT.md file to my project.